ingestion-beam/src/main/java/com/mozilla/telemetry/options/ErrorOutputType.java

```diff
@@ -5,19 +5,24 @@
 package com.mozilla.telemetry.options;
 
 import com.mozilla.telemetry.transforms.Println;
+import java.util.HashMap;
+import java.util.Map;
 import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessageWithAttributesCoder;
+import org.apache.beam.sdk.transforms.MapElements;
 import org.apache.beam.sdk.transforms.PTransform;
 import org.apache.beam.sdk.values.PCollection;
 import org.apache.beam.sdk.values.POutput;
+import org.apache.beam.sdk.values.TypeDescriptor;
 
 /**
- * Enumeration of error output types that each provide a {@code write} method.
+ * Enumeration of error output types that each provide a {@code writeFailures} method.
```

Comment:
We now have a concrete `write` method that calls the abstract `writeFailures` method. This allows us to consolidate logic for the new `includeStackTraces` configuration option.